working/wildcards/feature-specification.md

```diff
@@ -218,8 +219,18 @@ class C {
 }
 ```
 
+It is a compile-time error if a named initializing formal has the name `_`:
+
+```dart
+class C {
+  var _;
+  
+  C({this._}); // Error.
+}
+```
```

Comment:
This is not a new error, correct? This is just the existing error for private named parameters? If so, it might be good to clarify that here.

Reply:
Yeah, to confirm, this is just https://dart.dev/tools/diagnostic-messages#private_optional_parameter right?

Reply:
Right, good catch! I thought it was a new error because it can be motivated in terms of wildcard specific properties: A named parameter must have a name, but wildcard parameters do not.

However, we have [this](https://github.com/dart-lang/language/blob/bb23c2b8e12ddb54d485d53c8690d2d698c3cab1/specification/dartLangSpec.tex#L2412) in the language specification already:

> It is a compile-time error if the name of a named optional parameter begins with an `_' character.

(It is updated to say 'if the name of a named parameter' with the [null-safety related updates](https://github.com/dart-lang/language/pull/2605), because it applies to `required` named parameters as well as optional ones.)

If we adopt https://github.com/dart-lang/language/issues/2509 then formal parameters of the form `this._...` and `super._...` will presumably be allowed as long as there is a corresponding public name. But this still makes the bare `_` an error (a public name can't be empty).

I changed it to commentary.